machine-learning/house_price/regression.dig

```diff
@@ -32,13 +31,13 @@ _export:
 #     eta0: [5.0, 1.0, 0.5, 0.1, 0.05, 0.01, 0.001]
 
 #   _parallel: true
-  
+
```

Comment:
better to remove unused codes and working codes. 

Keep to it simple.

Reply:
Nice catch! Removed 7901591